tests/__init__.py

```diff
@@ -0,0 +1,191 @@
+"""
+__init__.py
+Date:  07/08/2020
+Author: Eugeniu Costetchi
+Email: costezki.eugen@gmail.com 
+"""
+from typing import Tuple, Optional
+
+import pandas as pd
+
+from eds.adapters.base_data_source import UnsupportedRepresentation, DataSource
+from eds.adapters.file_ds import FileDataSource
+from eds.adapters.sparql_ds import SPARQLEndpointDataSource
+from eds.builders.jinja_builder import FROM_ENDPOINT, FROM_FILE
+
+SPO_LIMIT_10 = "select * where {?s ?p ?o} limit 10"
+WRONG_SPO_LIMIT_10 = "select * "
+QUERY_LONGER_THAN_2048_CHARACTERS = "select * {" * 205
+LOCAL_CORRECT = "http://localhost:3030/subl"
+REMOTE_CORRECT = "http://publications.europa.eu/webapi/rdf/sparql"
+INEXISTENT_SERVER = "http://inexistent/mocked/server"
+CRASHED_SERVER = "http://crashed/mocked/server"
+
+RESPONSE_SPARQL_CSV_CORPORATE_BODY = [["s", "p", "o"],
```

Comment:
I think all these constants are better suited to be moved in the `conftest.py` file

Reply:
yeah, the morning discussion.

I did a test and only the pytest fixtures are nicely handled there.
will have to still wrangle with it in the future